doc/src/Configuration.md

```diff
@@ -54,7 +54,6 @@ customdnsrecords = [
     # list of locations to recursively read blocklists from (warning, every file found is assumed to be a hosts-file or domain list)
     sourcedirs = ["sources"]
     # manual blocklist entries
-    blocklist = []
     # manual whitelist entries - comments for reference
```

Comment:
why remove this? it is correct and part of the config. [See source](https://github.com/Cottand/leng/blob/master/config.go#L44)

Is the problem that the intent or meaning of the config param is not clear?

Reply:
ah I see it was a duplicate, fair enough. In that case please also remove the comment above it :pray: